lxd/db/cluster/devices.mapper.go

```diff
@@ -27,7 +27,7 @@ const deviceDelete = `DELETE FROM %s_devices WHERE %s_id = ?`
 
 // GetDevices returns all available devices for the parent entity.
 // generator: device GetMany
-func GetDevices(ctx context.Context, tx *sql.Tx, parent string) (map[int][]Device, error) {
+func GetDevices(ctx context.Context, tx *sql.Tx, parent string, filters ...DeviceFilter) (map[int][]Device, error) {
```

Comment:
I cant see where this function errors out of an empty filter is provided?

Reply:
Sorry, forgot these ones use a different section in the generator. Fixed now.